image_analysis/timetable_analysis.py

```
@@ -113,11 +113,15 @@ async def _queue_worker():
                 _THREAD_EXECUTOR, extract_schedule_fixed_scaled, img
             )
             job["status"] = JobStatus.DONE
+            print(f"[Worker DONE] job_id={job_id}, user_id={user_id}, result_count={len(result) if result else 0}")
             if result:
                 await _post_to_spring(user_id, job_id, result, token)
+            else:
+                print(f"[Worker] OCR result is empty, skipping Spring POST")
```

Ruff (0.15.4)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)